apps/portal/src/content/docs/components/button-options.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: ButtonWithOptions
+description: ButtonWithOptions component documentation
+---
+
+The `ButtonWithOptions` component provides a flexible UI element for displaying a button with a dropdown of options. It can behave as a radio button group if a `selectedValue` is provided, or as a regular dropdown otherwise.
+
+import { DocsPage } from "@/components/docs-page";
+import { ButtonWithOptions } from "@harnessio/ui/components";
+import { Button } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="space-y-4">
+    <div className="mb-4 text-white">
+      <div className="text-lg font-bold">Default Button (Default Size)</div>
```

Comment:
In general, try to use semantic tags. This is probably best as a paragraph or a header of some sort (semantically it would be an `h3`).